[New] Real-time web socket tutorial rewrite (#2408)
timescale/docs: _partials/_graphing-ohlcv-data.md

## Graph OHLCV data

When you have extracted the raw OHLCV data, you can use it to graph the result
in a candlestick chart, using Grafana. To do this, you need to have Grafana set
up to connect to your TimescaleDB database. For more information about how to do
this, see the [Grafana setup instructions][grafana-setup].

<Procedure>

### Graphing OHLCV data

> 1.  Ensure you have Grafana installed, and you are using the TimescaleDB
>     database that contains the Twelve Data stocks dataset set up as a
>     data source. For more information about how to do this, see the [Grafana
>     setup instructions][grafana-setup].

Reviewer: ```suggestion
1.  Install Grafana and the Twelve Data stocks database following the [Grafana
    setup instructions][grafana-setup].
```

Reviewer 2: @Rajakavitha1 It just occurred to me that we might have a partial we can use here. Check through the Grafana partials?

Author: Hi @Loquacity I checked the _grafana_vix_prereqs and it reads: Imported the stock trade data from the [Getting Started Tutorial][gsg-data] in this tutorial we are ingesting realtime with a web socket.

Reviewer 2: hrm, OK, no worries. Sorry for the bum steer ☹️

> 1.  In Grafana, from the `Dashboards` menu, click `New Dashboard`. In the
>     `New Dashboard` page, click `Add a new panel`.

Reviewer: ```suggestion
1.  Add a new panel for the tutorial. From the `Dashboards` menu in Grafana, click `New Dashboard` and then click `Add a new panel`.
```

Author: Style guide: The place of the action, then the action.

1.  In the `Visualizations` menu in the top right corner, select `Candlestick`
    from the list. Ensure you have set the Twelve Data stocks dataset as
    your data source.

> 1.  Click `Edit SQL` and paste in the query you used to get the OHLCV values.

Reviewer: Link to the query?

Author: This is a partial that is reused in two tutorials.

> 1.  In the `Format as` section, select `Table`.

Reviewer: Might be good to explain why first in the paragraph. I'm not sure what this does.

Author: No concepts in the task topics. This is a How to content and we explain only how to and not why in tasks topics. In the concept topics, we explain the why.
